EventListener/EmailSubscriber.php

```diff
@@ -10,7 +10,9 @@
 use Mautic\EmailBundle\Event\EmailSendEvent;
 use Firebase\JWT\JWT;
 
-$privateKey = <<<EOT
+// generate a random token
+function getToken() {
```

Comment:
thats not looks like a generate Token function more like an get this token and over. And why is the RSA Private key into code, i think this sould be read out of a file and not hard coded.

Reply:
the comment about generating a random token was from a previous version. I tried to get it to read the private key from an extra file, but I haven't gotten it to work. If you know how to properly do it, feel free to do it

Reply:
Mhm it is php, i think there are some file operations possible dont ask me :) But u can open / read a file in php and i think that would be the better way and dont commit this file to github, thats why it is called private :P

This should possible with: http://php.net/manual/de/function.readfile.php i think.